MCDataProducts/inc/KalSeedMC.hh

```diff
@@ -34,15 +34,17 @@ namespace mu2e {
     MCRelationship _rel; // relationship of this particle to its primary
     uint16_t _nhits; // number of associated StrawHits
     uint16_t _nactive; // number of associated active hits
-    XYZVectorF _mom; // initial momentum
+    CLHEP::HepLorentzVector   _mom; // initial momentum
```

Comment:
If you need the full Lorentz vector then use XYZTVectorF to store it https://root.cern/root/html606/namespaceROOT_1_1Math.html#a7a7702034ab0a4e722810fc0bccb5578